docker/wis2node/wis2node.cron

```diff
@@ -0,0 +1,9 @@
+### Clean out files 
+0 * * * * /usr/bin/find $WIS2NODE_DATADIR_INCOMING -type f -mtime +$WIS2NODE_DATADIR_RETENTION_DAYS -exec rm {} \; > /dev/null 2>&1
+0 * * * * /usr/bin/find $WIS2NODE_DATADIR_OUTGOING -type f -mtime +$WIS2NODE_DATADIR_RETENTION_DAYS -exec rm {} \; > /dev/null 2>&1
+0 * * * * /usr/bin/find $WIS2NODE_DATADIR_PUBLIC -type f -mtime +$WIS2NODE_DATADIR_RETENTION_DAYS -exec rm {} \; > /dev/null 2>&1
+
+### Clean out empty directories
+0 * * * * /usr/bin/find $WIS2NODE_DATADIR_INCOMING -type d -empty -delete > /dev/null 2>&1
```

Comment:
I'm  not sure we want to delete the incoming data at this stage, especially once we start pointing at the s3 bucket / ftp server. I would suggest adding an archive directory where the incoming data can be moved to once successfully processed.

Reply:
@david-i-berry that's fine... I just don't want an input directory to grow without bound... the most efficient input directory is one that is kept empty.   Putting the data somewhere else is fine.  could even just copy the input into public, and avoid creating another directory.

Reply:
Moving to an archive directory, zipping and taring would be my choice in the short term. In the longer term the appropriate archive for the incoming data needs to be identified.

Reply:
For now, I've commented out any removals from incoming.  For longer term discussion.